python/src/puml/generate_plantuml_diagrams_impl.py

```diff
@@ -23,6 +23,66 @@
 from .helpers.sequence_helpers import _get_use_case_participants, _get_use_case_steps
 
 plugin_name = "Generate PlantUML Diagrams"
+COMPONENT_STRING = "component"
+OBJECT_STRING = "object"
+SEQUENCE_STRING = "sequence"
+REQUIREMENTS_STRING = "requirements"
```

Comment:
```suggestion
```

These are no longer being used so they can go away.  I had them removed in mine.

Reply:
removed